### ISSUE TITLE
Convert Geodetic CS to another Geodetic CS with different  datum, but elevation value are not correct in proj6.1.0 version.

### ISSUE THREAD

**Issue by SamuraiBug** [OSGeo/PROJ#1487]
Hello，the problem as follow:
The Example:
cs2cs +proj=latlong +ellps=GRS80 +towgs84=-199.87,74.79,246.62
    +to +proj=latlong +datum=WGS84
20 35
20d0'5.467"E    35d0'9.575"N 8.570

I got the same result using Proj5.2.0，but in Proj6.1.0 it returns:
20d0'5.467"E    35d0'9.575"N 0.000

I guessed that elevation value could not be converted correctly in Proj6.1.0. Vertical grid shift failed?
Thanks.

**Comment by kbevers** [comment]
Where is the example from?

The result you see is not wrong, per se. In the 5.2 result you get an adjustment of the height, whereas that is discarded in 6.1. The reason being that when you are transforming from a geodetic CRS to another geodetic CRS we assume you want the height component to be unchanged, since you don't specify the height system of either the input or output coordinates. Actually, you don't even specify a height.

You can the result you expect by runnning this command:

```
echo 20 35 0 0 | cct +proj=pipeline +step +proj=cart +ellps=WGS84 +step +proj=helmert +x=-199.87 +y=74.79 +z=246.62 +step +proj=cart +ellps=WGS84 +inv
 20.0015187453   35.0026597383        8.5673        0.0000
```

To sum up, this is not a bug. cs2cs is acting as it was designed, but you can get a similar result to what you are asking for with cct.

**Comment by SamuraiBug** [comment]
The example from https://proj4.org/usage/transformation.html.
As a beginner, i need to find out more about features of proj4.
Thanks a lot.

**Comment by kbevers** [comment]
> The example from proj4.org/usage/transformation.html.

I figured. I'll update the docs right away. Thanks for reporting that error.
